### PR TITLE
Add trigger for go-ovirt-client-tests suite

### PR DESCRIPTION
**Pull request by emesika**
Adds a trigger file that will trigger execution in OST of the
go-ovirt-client-tests-suite-master suite

## Please describe the change you are making

...

## Are you the owner of the code you are sending in, or do you have permission of the owner?

...

## The code will be published under the BSD 3 clause license. Have you read and understood this license?

...

### CODE REVIEW

File: .github/workflows/go-ovirt-client-tests-suite-trigger.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: OST go-ovirt-client-tests-suite trigger
```
Comment (janosdebugs) [inline .github/workflows/go-ovirt-client-tests-suite-trigger.yaml:1]: Can we name this something a bit more readable? Like `OST trigger`?

**Review by janosdebugs — APPROVED** [review]
LGTM

File: .github/workflows/go-ovirt-client-tests-suite-trigger.yaml (hunk 1 above)
Comment (michalskrivanek) [reply .github/workflows/go-ovirt-client-tests-suite-trigger.yaml:1]: also, please use the current example, not old code...

**Comment by michalskrivanek** [comment]
obsoleted by #120

**Pull request closed without merging**